content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -300,7 +300,7 @@ fail validation.
 			<td><strong>Policy</strong></td>
 		</tr>
 		<tr>
-			<td colspan="2"><em>Everything from the baseline profile.</em></td>
+			<td colspan="2"><em>Everything from the baseline policy</em></td>
```

Comment:
I think keeping it as _profile_ is better so that the reader doesn't get confused.

Reply:
Hi @Ritikaa96 - I also prefer `profile `, however `policy` is (currently) the more common term on the page.  Happy to change all instances once we have a decision. :)